src/components/project/project.service.ts

```diff
@@ -212,8 +212,7 @@ export class ProjectService {
     const createdAt = DateTime.local();
     const createInput = {
       sensitivity: Sensitivity.High, // TODO: this needs to be calculated based on language engagement
-      step: ProjectStep.EarlyConversations,
-      status: stepToStatus(ProjectStep.EarlyConversations),
+      status: stepToStatus(input.step || ProjectStep.EarlyConversations),
```

Comment:
Shouldn't need it both here and in the DTO

Reply:
It's giving me a typeError because input.step can be undefined since it's an optional field in the DTO

Reply:
actually I'll add a !